apps/sms/views/shared/style/root.css

```diff
@@ -12,7 +12,7 @@ body {
   font-size: 1.6rem;
   line-height: 2.2rem;
   vertical-align: middle;
-  background-color: #FFF;
+  background: var(--header-background);
```

Comment:
Mmm, correct me if I'm wrong, but when we see white flickering - it's entire app frame is white, header is not even displayed, when we see the header - we basically already see ready app page. That is why I think we should use more generic theme color - the same as splash screen and status bar, rather than header. Am I missing something or we're talking about different flickering? :)